x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/export_rules/route.ts

```diff
@@ -65,10 +70,19 @@ export const exportRulesRoute = (
               body: `Can't export more than ${exportSizeLimit} rules`,
             });
           } else {
-            const nonPackagedRulesCount = await getNonPackagedRulesCount({
-              rulesClient,
-            });
-            if (nonPackagedRulesCount > exportSizeLimit) {
+            let rulesCount = 0;
+
+            if (prebuiltRulesCustomizationEnabled) {
+              rulesCount = await getRulesCount({
+                rulesClient,
+                filter: '',
```

Comment:
Does an empty filter string just mean fetch everything?

Reply:
Yep! It's a required argument and we typically call it with [a predefined filter for mutable rules](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/search/get_existing_prepackaged_rules.ts#L20-L26), but as this PR is extending that functionality this is a consequence of that. I could make it optional and default to `''` if it's not specified, if you prefer.

Reply:
Nah, it's a bit wonky looking but easy enough to understand what it's doing - I was just curious.